README.md

```diff
@@ -28,7 +28,7 @@ First, ensure your project is using [Go modules](https://blog.golang.org/using-g
 
 In your project directory, install the driver using go get:
 
-```go get github.com/awslabs/amazon-qldb-driver-go/qldbdriver```
+```go get github.com/awslabs/amazon-qldb-driver-go/qldbdriver/v2```
```

Comment:
I think we still need to support v1 qldbdriver aswell.

Reply:
v1 is no longer on the master branch.